tests/integration/test_logcollector/test_location/test_location.py

```diff
@@ -0,0 +1,182 @@
+               os.path.join(temp_dir, 'wazuh-testing', 'ñ.txt')],
+     'log_format': 'syslog', 'file_type': 'wildcard_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'Testing white spaces'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'Testing white spaces')], 'log_format': 'syslog',
+     'file_type': 'single_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'test.*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'test.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', 'test.log')],
+     'log_format': 'syslog', 'file_type': 'wildcard_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'c*test.txt'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'c1test.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', 'c2test.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', 'c3test.txt')], 'log_format': 'syslog',
+     'file_type': 'wildcard_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt')],
+     'log_format': 'syslog', 'path_2': os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt'),
+     'file_type': 'duplicated_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'file.log-%Y-%m-%d'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing',f'file.log-{date}')], 'log_format': 'syslog',
+     'file_type': 'single_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs', '*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs', 'multiple')],
+     'log_format': 'syslog', 'file_type': 'multiple_logs'}
+]
+
+if sys.platform != 'win32':
+    for case in metadata:
+        if case['location'] == os.path.join(temp_dir, 'wazuh-testing', '*'):
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'テスト.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'ИСПЫТАНИЕ.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', '测试.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'اختبار.txt'))
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def create_directory():
+    """Create expected directories."""
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs'), exist_ok=True)
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'depth1', 'depth2'), exist_ok=True)
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'duplicated'), exist_ok=True)
+    yield
+    rmtree(os.path.join(temp_dir, 'wazuh-testing'), ignore_errors=True)

```

Comment:
Maybe this fixture could be more general and add it to the integration conftest file.